README.md

```diff
@@ -53,7 +59,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n) because it should know where it is in the list using 2 pointers (start and end). This will tell you if it meets the requirements of the configuration as it iterates thorugh the list in O(n/2) -> O(n). 
```

Comment:
# -1

Yes, it's O(n). But the parser doesn't know the whole path. The parser takes its input one token at a time. It doesn't know where the end is
until it gets there. So the mechanical parsing process is to get the next token,
push it on a stack, and then compare the rest of the input. As a result, the
stack can grow to at least n/2, so the memory consumption will be O(n)